ui/page/privacy/manual_mixer_setup_page.go

```diff
@@ -122,65 +119,123 @@ func (pg *ManualMixerSetupPage) OnNavigatedTo() {
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
 func (pg *ManualMixerSetupPage) Layout(gtx layout.Context) layout.Dimensions {
-	body := func(gtx C) D {
-		page := components.SubPage{
-			Load:       pg.Load,
-			Title:      values.String(values.StrManualSetUp),
-			BackButton: pg.backButton,
-			Back: func() {
-				pg.ParentNavigator().CloseCurrentPage()
-			},
-			Body: func(gtx C) D {
-				return pg.Theme.Card().Layout(gtx, func(gtx C) D {
-					gtx.Constraints.Min.X = gtx.Constraints.Max.X
-					return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
-						layout.Flexed(1, func(gtx C) D {
-							return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+	return layout.Inset{Top: values.MarginPadding40}.Layout(gtx, func(gtx C) D {
+		return pg.Theme.Card().Layout(gtx, func(gtx C) D {
+			gtx.Constraints.Min.X = gtx.Constraints.Max.X
+			return layout.Flex{Axis: layout.Vertical, Alignment: layout.Start}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
```

Comment:
This lines are not needed and seem repeated. I would recommend using the linear Layout widget for this though.

Reply:
Oh yeah, was able to remove the `Flex` layout here and it looks the same.

Linear in place of the Center below or the Card above?

Reply:
@JoeGruffins  I think it just:

```
return layout.Flex{Alignment: layout.Start}.Layout(gtx,
					layout.Rigid(func(gtx C) D {
						gtx.Constraints.Max.X = int(values.MarginPadding40)
						return layout.Center.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
							return pg.Theme.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
						})
					}),
					layout.Rigid(func(gtx C) D {
						return layout.Inset{
							Left: values.MarginPadding10,
						}.Layout(gtx, func(gtx C) D {
							return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
								layout.Rigid(func(gtx C) D {
									label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleWarningTitle))
									return label.Layout(gtx)
								}),
								layout.Rigid(func(gtx C) D {
									label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleWarningDesc))
									return label.Layout(gtx)
								}),
							)
						})
					}),
				)
```

this is my suggestion code.